assets/admin.html

```diff
@@ -1,6 +1,13 @@
 <html><head>
     <title>Hull Clearbit</title>
     <link rel="stylesheet" href="//dd04rofzygnm0.cloudfront.net/releases/master/865f04865d2448286626bac92c518a8f8ea8bafe/stylesheets/neue.css">
+    <link href="https://cdnjs.cloudflare.com/ajax/libs/select2/4.0.3/css/select2.min.css" rel="stylesheet" />
+
+    <script
+      src="//code.jquery.com/jquery-2.2.4.min.js"
```

Comment:
@michaloo we cannot use external dependencies. It would leak the connector credentials